app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/components/summary/EntryNumberSummary.scala

```diff
@@ -17,14 +17,15 @@
 package uk.gov.hmrc.cdsreimbursementclaimfrontend.views.components.summary
 
 import play.api.i18n.Messages
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.{JourneyBindable, routes}
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.ReimbursementRoutes.ReimbursementRoutes
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.routes
 import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.ids.EntryNumber
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
-import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{ActionItem, Actions, Key, SummaryList, SummaryListRow, Value}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist._
 
 class EntryNumberSummary extends AnswerSummary[EntryNumber] {
```

Comment:
we don't need this anymore, right? Entry number is dead.

Reply:
Yes decided to leave it as I have something like this right now
```
        @summary(completeClaim.movementReferenceNumber.value.fold(_.review(s"$key.entry-reference-number"), _.review(s"$key.mrn")))
```
Removing it I need somehow handle scenario
```
        @summary(completeClaim.movementReferenceNumber.value.map(_.review(s"$key.entry-reference-number")).valueOr(//What to do here, throw exception via fail ? )
```
So felt perhaps best to delete it together with EntryNumber, as no such exceptional error handling will be required

Reply:
I went bold with `.right.get`

Reply:
The issue with doing an exception is that we might forget about it when we eventually rip out the entry number stuff so leave it here now and when we delete that model the compiler should tell us to remove that anyway.